.github/workflows/release.yaml

```diff
@@ -222,7 +222,7 @@ jobs:
       - name: Login to the OCI registries
         run: |
           echo "${{ secrets.QUAY_DEPLOYER_PASSWORD }}" | helm registry login quay.io --username "${{ vars.QUAY_DEPLOYER_USERNAME }}" --password-stdin
-          echo "${{ github.token }}" | helm registry login ghcr.io --username "${{ github.actor }}" --password-stdin
+          echo "${{ github.token }}" | helm registry login ghcr.io --username $ --password-stdin
```

Comment:
What's the expected behavior of the dollar sign in `--username $` ?

Reply:
I can't find it documented - I just saw someone use it in a workflow, hence my comment https://github.com/kata-containers/kata-containers/pull/11331#issuecomment-2919327280

Reply:
Interesting - maybe this was my big chance for a fun Easter egg!

Reply:
Thanks @stevenhorsman ! Would you mind adding this clarification to the commit message ?